### ISSUE TITLE
Placeholder ticket for Pubvendors.json Vs Vendors.txt support

### ISSUE THREAD

**Issue by anissaconnor** [digi-trust/cmp#105]
This is a single ticket to encompass the work associate with supporting pubvendors.json. The spec is currently open for public comment so awaiting to see the outcome prior to starting dev work to support this. Details on likely changes required are documented here: https://docs.google.com/document/d/1DXadVaKgx2ygDMavDwUGTF--ViWpFjrp1dqp6MCSRd0/edit#

**Comment by SupahNickie** [comment]
Hi Kevin,

Thanks for reaching out. We currently have almost the entire implementation already done, but are waiting on the IAB to finalize the specification for the new format the encoded vendor cookie is going to take before we can wrap everything up. As soon as they finalize that cookie specification (should be within a couple of weeks), this will be our top priority and we should be in a good place to get that new functionality up rather quickly.
